tests/unit/jest.config.js

```diff
@@ -41,6 +43,11 @@ module.exports = {
   ],
   ...conditionalWebComponentsConfig,
   coverageDirectory: `<rootDir>/tests/unit/coverage-data`,
+  reporters: [
+    'default',
+    ['<rootDir>/tests/unit/testsReporter.js', { type }],
```

Comment:
adding this reporter to summarize the total number of failed, skipped, passed tests when running unit tests. That json is saved in `tests/unit/coverage-data`

cc: @tamara-corbalt